fiboa_cli/datasets/ec_si.py

```diff
@@ -0,0 +1,189 @@
+"""
+# Bounding box of the data in WGS84 coordinates
+BBOX = [13.410659993139568, 45.42227046875208, 16.596303417885895, 46.87424125552635]
+
+# Provider name, can be None if not applicable, must be provided if PROVIDER_URL is provided
+PROVIDER_NAME = "Eurocrops"
+# URL to the homepage of the data or the provider, can be None if not applicable
+PROVIDER_URL = "https://github.com/maja601/EuroCrops/wiki/Slovenia"
+# Attribution, can be None if not applicable
+ATTRIBUTION = "Ministrstvo za kmetijstvo, gozdarstvo in prehrano"
+
+# License of the data, either
+# 1. a SPDX license identifier (including "dl-de/by-2-0" / "dl-de/zero-2-0"), or
+LICENSE = "CC-BY-4.0"
+# 2. a STAC Link Object with relation type "license"
+# LICENSE = {"title": "CC-BY-4.0", "href": "https://creativecommons.org/licenses/by/4.0/", "type": "text/html", "rel": "license"}
+
+# Map original column names to fiboa property names
+# You also need to list any column that you may have added in the MIGRATION function (see below).
+COLUMNS = {
+    'geometry': 'geometry', #fiboa core field
+    'ID': 'id', #fiboa core field
+    'AREA': 'area', #fiboa core field
+    'GERK_PID': 'gerk_pid', #fiboa custom field
+    'SIFRA_KMRS': 'crop_type_class', #fiboa custom field
+    'RASTLINA': 'rastlina', #fiboa custom field
+    'CROP_LAT_E': 'crop_lat_e', #fiboa custom field
+    'COLOR': 'color', #fiboa custom field
+    'EC_trans_n': 'EC_trans_n', #fiboa custom field
+    'EC_hcat_n': 'EC_hcat_n', #fiboa custom field
+    'EC_hcat_c': 'EC_hcat_c' #fiboa custom field
+}
+
+# Add columns with constant values.
+# The key is the column name, the value is a constant value that's used for all rows.
+ADD_COLUMNS = {
+    "determination_datetime": "2021-10-06T00:00:00Z"
+}
+
+# A list of implemented extension identifiers
+EXTENSIONS = []
+
+# Functions to migrate data in columns to match the fiboa specification.
+# Example: You have a column area_m in square meters and want to convert
+# to hectares as required for the area field in fiboa.
+# Function signature:
+#   func(column: pd.Series) -> pd.Series
+COLUMN_MIGRATIONS = {}
+
+# Filter columns to only include the ones that are relevant for the collection,
+# e.g. only rows that contain the word "agriculture" but not "forest" in the column "land_cover_type".
+# Lamda function accepts a Pandas Series and returns a Series or a Tuple with a Series and True to inverse the mask.
+COLUMN_FILTERS = {}
+
+# Custom function to migrate the GeoDataFrame if the other options are not sufficient
+# This should be the last resort!
+# Function signature:
+#   func(gdf: gpd.GeoDataFrame) -> gpd.GeoDataFrame
+MIGRATION = None
+
+# Schemas for the fields that are not defined in fiboa
+# Keys must be the values from the COLUMNS dict, not the keys
+MISSING_SCHEMAS = {
+    'required': ['gerk_pid', 'crop_type_class', 'rastlina', 'crop_latin', 'color', 'EC_trans_n','EC_hcat_n', 'EC_hcat_c'], # i.e. non-nullable properties
```

Comment:
```suggestion
    'required': ['gerk_pid', 'crop_type_class', 'rastlina', 'crop_latin', 'color', 'EC_trans_n','EC_hcat_n'], # i.e. non-nullable properties
```

to fix https://github.com/fiboa/cli/pull/45/files#r1634625809